### ISSUE TITLE
Incompatible with materialize:materialize ?

### ISSUE THREAD

**Issue by rjalili** [alethes/meteor-pages#197]
The pagination nav seems to have an issue when the materialize package is being used.

Looks like the problem is a conflict in the css for the li element.

**Comment by slfrsn** [comment]
Agreed. `.pagination` is a pretty generic class name. Could you (the devs) consider a more unique alternative? The work we have to go through to override the rules to work with Materialize is agonizing. Or give us the option of disabling your CSS. Whichever's easier.
